server/test/services/applicant/question/DateQuestionTest.java

```diff
@@ -88,4 +89,29 @@ public void withMisformattedDate() {
                         MessageKey.DATE_VALIDATION_INVALID_DATE_FORMAT))));
     assertThat(dateQuestion.getDateValue().isPresent()).isFalse();
   }
+
+  @Test
+  public void getDateValue_returnsPAIValueWhenTagged() {
+    DateQuestionDefinition dateQuestionDefinitionWithPAI =
+        new DateQuestionDefinition(
+            QuestionDefinitionConfig.builder()
+                .setName("question name")
+                .setDescription("description")
+                .setQuestionText(LocalizedStrings.of(Locale.US, "question?"))
+                .setQuestionHelpText(LocalizedStrings.of(Locale.US, "help text"))
+                .setId(OptionalLong.of(1))
+                .setLastModifiedTime(Optional.empty())
+                // Tag the question as a PAI question
+                .setPrimaryApplicantInfoTags(ImmutableSet.of(PrimaryApplicantInfoTag.APPLICANT_DOB))
+                .build());
+
+    // Save applicant's dob to the PAI column
+    applicant.setDateOfBirth("2001-01-01");
+
+    DateQuestion dateQuestion =
+        new ApplicantQuestion(dateQuestionDefinitionWithPAI, applicantData, Optional.empty())
+            .createDateQuestion();
+
+    assertThat(dateQuestion.getDateValue().get()).isEqualTo(applicant.getDateOfBirth().get());
```

Comment:
The dob stored in the database is already present on the question, without the applicant filling anything out.